resources/assets/coffee/react/profile-page/extra.coffee

```diff
@@ -19,21 +19,57 @@
 el = React.createElement
 
 class ProfilePage.Extra extends React.Component
+  # Mirrored from App/Models/User.php
+  # The numbers corresponding to particular profile section.
+  USER_PAGE = 1
```

Comment:
now I look again, why even bother having an integer constant mapping and not just use the section id string everywhere?